src/metabase/actions.clj

```diff
@@ -285,6 +285,9 @@
 
 ;;;; Bulk actions
 
+(s/def :actions.args.crud.bulk/table-id
+  :actions.args/id)
```

Comment:
It the expectation is that all bulk actions are going to require `:table-id`, I think you should call this `:actions.args.crud.bulk.common/table-id` and make a `:actions.args.bulk/common` spec like we have for `:actions.args.crud.common`. Then `:actions.args.crud.bulk/create` and the like can `s/merge` `:actions.args.bulk/common` instead of repeating `:actions.args.crud.bulk/table-id` inside of `s/keys` several times

Reply:
To be consistent with `row` type specs, I kept your suggestions but changed:
`:actions.args.bulk/common` to be `:actions.args.crud.bulk/common`

Reply:
Yeah that makes perfect sense. Thanks